cmd/world/main.go

```diff
@@ -35,5 +60,36 @@ func main() {
 		}()
 	}
 
+	// Posthog Initialization
+	posthog.Init(PosthogApiKey)
+	defer posthog.Close()
+
+	// Obtain the machine ID
+	machineID, err := machineid.ProtectedID("world-cli")
+	if err != nil {
+		logger.Error(err)
+	}
+
+	// Create capture event for posthog
+	event := ph.Capture{
+		DistinctId: machineID,
+		Timestamp:  time.Now(),
```

Comment:
it's probably better to just abstract away the `ph.Capture` object creation under `posthog.CaptureEvent`

therefore, you only need to pass in a string that represents the event and the cli context (containing the version/command), i.e. `posthog.CaptureEvent(ctx, runningEvent)`

Reply:
updated